client/remotemodel.cpp

```diff
@@ -390,13 +390,20 @@ void RemoteModel::newMessage(const GammaRay::Message &msg)
                 qWarning() << "Unexpected empty index, probably some type failed to deserialize" << Q_FUNC_INFO;
                 continue;
             }
+            quint32 indexEndPos = 0;
+            msg >> indexEndPos;
```

Comment:
Is this streamed size (redundant with where it actually is in the stream) needed? Wouldn't `int indexEndPos = payload().device()->pos();` do the job maybe?

Reply:
Indeed, thanks.